dev-python/neovim-remote/neovim-remote-2.1.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{4,5,6} )
+
+inherit distutils-r1
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mhinz/${PN}.git"
+else
+	KEYWORDS="~amd64"
+	SRC_URI="https://github.com/mhinz/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+fi
+
+DESCRIPTION="A tool that helps control neovim processes"
+HOMEPAGE="https://github.com/mhinz/neovim-remote"
+LICENSE="MIT"
+SLOT="0"
+IUSE="test"
+
+RDEPEND="
+	dev-python/neovim-python-client[${PYTHON_USEDEP}]
+	dev-python/psutil[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
+"
+
+DEPEND="${RDEPEND}
+	test? ( dev-python/pytest[${PYTHON_USEDEP}] )
+"
+
+python_test() {
+	py.test -v
```

Comment:
pytest -vv || die

Reply:
Added the `die`, but didn't notice a difference between `-v` and `-vv` output.

Reply:
I think it mostly affects skipped tests or sth.

Reply:
Also the dot is deprecated.